tests/fixture/wrangler_toml.rs

```diff
@@ -57,26 +67,76 @@ pub struct WranglerToml<'a> {
 }
 
 impl WranglerToml<'_> {
-    pub fn webpack_no_config(name: &str) -> WranglerToml {
+    pub fn webpack(name: &str) -> WranglerToml {
         let mut wrangler_toml = WranglerToml::default();
         wrangler_toml.name = Some(name);
-        wrangler_toml.workers_dev = Some(true);
         wrangler_toml.target_type = Some("webpack");
 
+        eprintln!("{:#?}", &wrangler_toml);
+        wrangler_toml
+    }
+
+    pub fn webpack_zoneless(name: &str, is_workers_dev: bool) -> WranglerToml {
+        let mut wrangler_toml = WranglerToml::webpack(name);
+        wrangler_toml.workers_dev = Some(is_workers_dev);
+
+        eprintln!("{:#?}", &wrangler_toml);
+        wrangler_toml
+    }
+
+    pub fn webpack_zoned_single_route<'a>(
```

Comment:
why do we need lifetimes for this? seems like there are other functions that take in regular `&str`

Reply:
if you pass in more than one &str you need the lifetimes.